docs/wiki/developer-guide.md

```
@@ -162,16 +154,16 @@ Entity spawn positions are defined in TOML level files in the `levels/` director
+[[coins]]
 x = 120.0
 y = 180.0
 
-[coins]
+[[coins]]
 x = 200.0
 y = 140.0
 ```
 
-Then extend `level_loader.c` to parse the new array table and call your `_init` function for each entry. See `level_design` for the full TOML schema and [Level Design — TOML Reference](#level-design) for placement examples for every entity type.
+Then extend `level_loader.c` to parse the new array table and populate the `GameState` array (or call your `_init` function when the entity owns richer runtime state). See `level_design` for the full TOML schema and [Level Design — TOML Reference](#level-design) for placement examples for every entity type.
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)